Workbooks/SapMonitor/SapNetWeaver/SapNetWeaver.workbook

```diff
@@ -67,6 +88,44 @@
       },
       "name": "overview_section_params"
     },
+    {
+      "type": 11,
+      "content": {
+        "version": "LinkItem/1.0",
+        "style": "nav",
+        "links": [
+          {
+            "id": "2d20d63e-d352-46e8-bcb0-9ca9ce897996",
+            "linkTarget": "WorkbookTemplate",
+            "linkLabel": "Alerts",
+            "style": "primary",
+            "workbookContext": {
+              "componentIdSource": "workbook",
+              "resourceIdsSource": "workbook",
+              "templateIdSource": "static",
+              "templateId": "Community-Workbooks/SapMonitor/Alerts/Alerts",
+              "typeSource": "workbook",
+              "gallerySource": "workbook",
+              "locationSource": "default",
+              "passSpecificParams": true,
+              "templateParameters": [
+                {
+                  "name": "ProviderType",
+                  "source": "parameter",
+                  "value": "param_provider_type"
+                },
+                {
+                  "name": "AlertTemplates",
+                  "source": "parameter",
+                  "value": "param_alert_templates"
+                }
+              ]
+            }
+          }
+        ]
+      },
+      "name": "links - 22"
```

Comment:
Can we provide some meaningful name here like AlertButton or AlertLink

Reply:
As it is step name that is propagated here, changed it **Alert Button Step**